index.html

```diff
@@ -10,10 +10,115 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="header">
+        <a href="#">
+            <img
+              src="img/karma-logo.svg"
+              alt="Larma Logo"
+              class="logo_header"
+            />
+        </a>
+        <nav>
+            <ul class="navigation_list">
+              <li class="navigation_item">
+                <a class="navigation_link" href="#">Meet Karma</a>
+              </li>
+              <li class="navigation_item">
+                <a class="navigation_link" href="#">How it Works</a>
+              </li>
+              <li class="navigation_item">
+                <a class="navigation_link" href="#">Store</a>
```

Comment:
The href on Store could be used to link the pages. I could not get it to flick between pages when I opened the files.